api/can_api_v2_definition.py

```diff
@@ -121,6 +121,19 @@ class Actuals(base_model.APIBaseModel):
         description="""
 Information about acute bed utilization details.
 
+Fields:
+ * capacity - Current staffed acute bed capacity.
+ * currentUsageTotal - Total number of acute beds currently in use
+ * currentUsageCovid - Number of acute beds currently in use by COVID patients.
+""",
+    )
+    hsaHospitalBeds: Optional[HospitalResourceUtilization] = pydantic.Field(
+        ...,
+        description="""
+Information about acute bed utilization details aggregated for the corresponding
+Health Service Area (HSA). For more on HSAs see: 
```

Comment:
Maybe say something about it only being available for county endpoints?

Reply:
Good call, added in 49dee09bda7dc6ad9c2f5c9a0c1f2fa37a6dec8c